src/actions/enter.ts

```diff
@@ -9,14 +9,12 @@ import {
   Events,
   ShortcutRegistry,
   utils as BlocklyUtils,
-} from 'blockly/core';
-
-import type {
   Block,
   BlockSvg,
   Field,
   FlyoutButton,
   WorkspaceSvg,
+  getFocusManager,
```

Comment:
Should not need to change all of the `import type`s to `imports` just to add the focus manager.

Reply:
Oops, I think I missed the type specifier and thought there were redundant import lines (which, in retrospect, I now realize would've been caught by the linter). Fixed in latest.